doc/installation.md

```diff
@@ -88,31 +95,20 @@ OR
 Install parallel-bilby:
 
 
-* $ conda install -c conda-forge parallel_bilby
+* $ conda install -c conda-forge parallel-bilby
+ 
 
+OR
 
 
 * $ pip install parallel-bilby
 
 
 
-Install pymultinest
```

Comment:
Why did this section disappear?

Reply:
Do you mean that the warning error ?
Base on our last discussion https://github.com/nuclear-multimessenger-astronomy/nmma/pull/45#discussion_r912408667  

NOTE: In case if an error comes up during an NMMA analysis of the form:
ERROR:   Could not load MultiNest library "libmultinest.so"
ERROR:   You have to build it first,
ERROR:   and point the LD_LIBRARY_PATH environment variable to it!
Then, for using the PyMultinest library, it is required to get and compile the Multinest library separately. Instructions for the same are given here:
https://johannesbuchner.github.io/PyMultiNest/install.html